quickstarts/microsoft.databasewatcher/watcher-new-datastore/azuredeploy.parameters.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "GEN-UNIQUE"
+        },
+        "location": {
+            "value": "East US"
+        },
+        "identityType": {
+            "value": "SystemAssigned"
+        },
+        "createNewDatastore": {
+            "value": true
+        },
+        "clusterName": {
+            "value": "GEN-UNIQUE"
+        },
+        "databaseName": {
+            "value": "GEN-UNIQUE"
+        },
+        "kustoOfferingType": {
+            "value": "adx"
+        },
+        "clusterSkuName": {
+            "value": "Standard_E2d_v5"
+        },
+        "clusterSkuTier": {
+            "value": "Standard"
+        },
+        "databaseWatcherApiVersion": {
+            "value": "2023-09-01-preview"
+        },
+        "targetCount": {
+            "value": 1
+        },
+        "targetProperties": {
```

Comment:
if you address my other comment and don't pass the entire properties body like this, then you can still use `GEN-UNIQUE` as you were before.